website/docs/r/dnat.html.markdown

```diff
@@ -15,21 +15,38 @@ and delete destination NATs to map an external IP/port to an internal IP/port.
 
 ```hcl
 resource "vcd_dnat" "web" {
+  org              = "my-org" #Optional
+  vdc              = "my-vdc" #Optional
+  
   edge_gateway = "Edge Gateway Name"
   external_ip  = "78.101.10.20"
   port         = 80
   internal_ip  = "10.10.0.5"
   translated_port = 8080
 }
+
+resource "vcd_dnat" "forIcmp" {
+  org              = "my-org" #Optional
+  vdc              = "my-vdc" #Optional
+  
+  edge_gateway = "Edge Gateway Name"
+  external_ip  = "78.101.10.20"
+  port         = -1
+  internal_ip  = "10.10.0.5"
+  protocol = "icmp"
+  icmp_sub_type = "router-solicitation"
+} 
 ```
 
 ## Argument Reference
 
 The following arguments are supported:
-
 * `edge_gateway` - (Required) The name of the edge gateway on which to apply the DNAT
 * `external_ip` - (Required) One of the external IPs available on your Edge Gateway
-* `port` - (Required) The port number to map
+* `port` - (Required) The port number to map. -1 translates to "any"
+* `translated_port` - (Optional) The port number to map
 * `internal_ip` - (Required) The IP of the VM to map to
+* `protocol` - (Optional; *v2.0+*) The protocol type. Possible values TCP, UDP, TCPUDP, ICMP, ANY. TCP is default to be back compatible with previous version
+* `icmp_sub_type` - (Optional; *v2.0+*) The name of ICMP type.
```

Comment:
Is there a supported list of values for icmp_sub_type to reference? Like in example you have "router-solicitation", where did you find it?

Reply:
There isn't explicit normal documentation for that